src/components/Editor.vue

```diff
@@ -717,6 +717,13 @@ export default {
 				return
 			}
 
+			if (event.key === 'Tab' && !event.shiftKey && this.$editor.isActive('codeBlock')) {
```

Comment:
Wouldn't this be something we could add to the CodeBlock node directly as a keyboard handler instead of here globally?

Reply:
@juliushaertl I have tried that way but it didn't work since tab key event was stoped by focus-trap js.